package.json

```diff
@@ -28,10 +28,12 @@
     "chai": "^3.5.0",
     "chai-as-promised": "^6.0.0",
     "coffee-script": "~1.11.0",
+    "fetch-mock": "^5.6.2",
     "gulp": "^3.8.10",
     "gulp-coffee": "^2.2.0",
     "gulp-mocha": "^3.0.0",
     "gulp-util": "^3.0.1",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
The idea for low-level modules is that we just tell the user `fetch` should be provided and expect them to include it on their own.
The reason is because this module add a global variable and I believe no deep dependencies (like our module) should do that to you behind the scene.
We include it in devDeps here just to be able to run tests